### ISSUE TITLE
Driver Crash And Restart After Prolonged Time

### ISSUE THREAD

**Issue by Xbye** [Xaymar/OBS-AMD-Advanced-Media-Framework#106]
The log is 165 MB, not able to upload it on pastebin for OBS github but if you really want the log I can upload it via mediafire or something. I am essentially streaming and then all of a sudden my computer freezes up, sound loops, both monitors go black and restart with OBS crashed/dead. Temperatures are monitored and never go above 70-80c.

**EDIT**: Uploaded log - https://www.mediafire.com/?s30x0aamww00bx9

This was the last bit of the LOG:

```
23:31:33.688: [AMFEncoderVCE]    Debug: QueryOutput(-1) Repeat: inputFrames=497439, outputFrames=497434
23:31:33.688: [AMFEncoderVCE]    Trace: SubmitInput() End: frame pts=497443 inputFrames=497440, TaskID=497442
23:31:33.688: [Plugin::GetOutput]    Trace: Packet: Type(1), PTS(497437), DTS(497435), Size(      33)
23:31:33.688: [AMFEncoderVCE]    Trace: SubmitInput() End: frame pts=497444 inputFrames=497441, TaskID=497443
23:31:33.688: [AMFEncoderVCE]    Debug: QueryOutput(-1) Repeat: inputFrames=497441, outputFrames=497434
23:31:33.688: [AMFEncoderVCE]    Debug: QueryOutput(-1) Repeat: inputFrames=497441, outputFrames=497434
23:31:40.468: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:31:44.948: [game-capture: 'Game Feed'] ----------------- d3d9 capture freed -----------------
23:31:45.222: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:31:45.222: Couldn't create UV vertex buffer.
23:31:47.301: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:31:47.627: [game-capture: 'Game Feed'] d3d9 shared texture capture successful
23:31:47.634: gs_texture_open_shared (D3D11): Failed to open resource (887A0005)
23:31:47.634: [game-capture: 'Game Feed'] init_shtex_capture: failed to open shared handle
23:31:47.635: [game-capture: 'Game Feed'] attempting to hook process: left4dead2.exe
23:31:47.681: [game-capture: 'Game Feed'] d3d9 shared texture capture successful
23:31:47.697: gs_texture_open_shared (D3D11): Failed to open resource (887A0005)
23:31:47.697: [game-capture: 'Game Feed'] init_shtex_capture: failed to open shared handle
23:31:50.322: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:31:50.322: Couldn't create UV vertex buffer.
23:31:51.655: [game-capture: 'Game Feed'] attempting to hook process: left4dead2.exe
23:31:55.426: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:31:55.426: Couldn't create UV vertex buffer.
23:32:00.531: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:00.531: Couldn't create UV vertex buffer.
23:32:05.635: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:05.635: Couldn't create UV vertex buffer.
23:32:10.530: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:10.551: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:10.614: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:10.637: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:10.739: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:10.739: Couldn't create UV vertex buffer.
23:32:10.926: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:10.947: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:11.280: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:11.322: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:11.447: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:11.468: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:15.844: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:15.844: Couldn't create UV vertex buffer.
23:32:15.864: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:15.884: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:15.905: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:15.926: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:16.009: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:20.947: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:20.947: Couldn't create UV vertex buffer.
23:32:24.822: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:24.884: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:24.905: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:24.926: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:24.947: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:25.134: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:25.260: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:26.894: ==== Shutting down ==================================================
23:32:28.072: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:28.113: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:28.134: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:28.468: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:28.488: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:29.780: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:29.822: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:30.114: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:30.135: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:30.135: device_vertexbuffer_create (D3D11): Failed to create buffer (887A0005)
23:32:30.135: Couldn't create UV vertex buffer.
23:32:31.072: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:31.114: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:31.155: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:31.176: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.239: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.280: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.322: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.572: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.592: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.613: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:33.697: device_texture_create (D3D11): Failed to create 2D texture (887A0005)
23:32:35.009: [game-capture: 'Game Feed'] capture window no longer exists, terminating capture
```

**Comment by Xbye** [comment]
@Xaymar 
Updated original post with mediafire link.

**Comment by Xaymar** [comment]
Known issue with the Anniversary Update: https://obsproject.com/forum/threads/psa-for-windows-10-anniversary-update-users.55256/

I managed to fix it locally by updating to the latest windows version, then using Display Driver Uninstaller in Safe Mode, Rebooting, signing in and waiting a few minutes for Windows Update to do it's stuff, Reboot, and then finally install 16.9.1. 16.9.2 does not seem to work well yet, but you can use the AMF runtime in it through other means (see FAQ).

**Comment by Xbye** [comment]
Issue _seems_ to be solved with the new hotfix driver, 16.10.1 after 2 hours 29min 30sec of streaming on Windows 10 (1601 Build).
